tests/parser/Eclipse3DPropertiesTests.cpp

```diff
@@ -537,10 +537,22 @@ COPY
   'PERMX' 'PERMY' /
 /
 
+EDIT
+
 MULTIPLY
   'PERMZ' 0.1D0 /
-  'PERMX' 0.1D0 *  *  1  21  *  1 / -- This is a weird way to specify the top layer! 
+  'PERMX' 0.1D0 *  *  1  21  *  1 / -- This is a weird way to specify the top layer!
+  'TRANX' 0.10  /
 /
+
+COPY
+  'TRANX' 'TRANY' /
+/
+
```

Comment:
How is the COPY keyword implemented? 

What happens if say tranx = 10 and trany = 100 
by default. And then tranx i multiplied with 0.1 before 
trany is set to tranx as in this example?

Reply:
hm: this is totally unrelated to the current PR, but does the current implementation of COPY work "correctly" if the two fields do exhibit different units? e.g:

```
TEMPI
100*10 /

COPY
TEMPI PERMX /
/
```

If I understand this correctly, PERMX and TEMP should exhibit the same values in whatever unit system the deck uses here. (yeah, this is probably not relevant in practice, but IMO it should be fixed.)

Reply:
The case @andlaus point will fail - silently. It can of course be fixed; they came to the moon afterall, but it would be a semi significant undertaking. The problem you address is in my opinion not very relevant - not something I would embark on before $$ talks.

Reply:
> How is the COPY keyword implemented?

Quite naively; the only relevant point in this context is that the `GridProperty->assigned` property is carried over in the copy process; i.e. if one is default initialized then the other will also have the flag `->assigned` set to `false`

> What happens if say tranx = 10 and trany = 100
by default. And then tranx i multiplied with 0.1 before
trany is set to tranx as in this example?

Quite frankly I do not care; that is just to contrived to be interesting. But of course - in general it will not be difficult to find failure modes, I am sure you can find others.

Reply:
> Quite frankly I do not care
You know better than me what kind of usage we could expect from reservoir engineers. I am just trying to understand how the code works.